dmutils/content_loader.py

```diff
@@ -387,6 +387,30 @@ def _get_single_question_data(self, form_data):
 
         if self.id == 'serviceTypes' or self.type in ['list', 'checkboxes']:
             value = form_data.getlist(self.id)
+        elif self.type == 'boolean_list':
+            value = []
+
+            # if self.id is 'q5', form values will come back as 'q5-0', 'q5-1', 'q5-3', ...
+            # here, we build a list of the values after the hyphen, eg [0, 1, 3, ...]
+            boolean_list_indexes = [
```

Comment:
We could build a `{index: value}` mapping here and then we won't have to rebuild the key again.
Building the list then becomes:
```python
value = [None] * max(boolean_indexes.keys() + 1)
for k, v in boolean_indexes.iteritems():
    value[k] = v
```